packages/ui-core/theme/src/lib/components/theme-sidebar/theme-settings/components/layout-selector/layout-selector.component.html

```
@@ -24,16 +24,16 @@
             >
           }
         </nb-select>
+        <button
+          class="reset-layout"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)